uiowa/uids
# Feature: additional style updates for brand.uiowa.edu and uiowa.edu

Contains updates related to https://github.com/uiowa/uiowa/issues/1557. 

# How to test

https://github.com/uiowa/uids/issues/190

- Go to http://localhost:3000/components/detail/card--default and add a list item to the card body and verify that it is same font size as a `<p>` tag.  

https://github.com/uiowa/uids/issues/188

- Add `<p><a href="/" class="bttn bttn--link bttn--sans-serif">View All Stories <i class="fas fa-arrow-right"></i></a> | <a href="/" class="bttn bttn--link bttn--sans-serif">View All News <i class="fas fa-arrow-right"></i></a></p>`under "Featured Stories" on http://localhost:3000/components/detail/uiowa and verify that it will display on two lines. 


https://github.com/uiowa/uids/issues/187

- Verify line-height on h2-h6 is set to 1.25 on http://localhost:3000/components/detail/headings

https://github.com/uiowa/uids/issues/186

- Test overlap highlight text on http://localhost:3000/components/detail/portraits-block and verify that it looks similar to the screenshot below. 

<img width="263" alt="Screen Shot 2020-05-18 at 2 32 57 PM" src="https://user-images.githubusercontent.com/1036433/82252918-93210880-9915-11ea-9bd5-163463f89959.png">


https://github.com/uiowa/uids/issues/181

- Verify that box shadow on bold underlined heading on http://localhost:3000/components/detail/portraits-block will adjust from white to gray if `uids-component--gray` class is added to container. 


https://github.com/uiowa/uids/issues/180

- Go to http://localhost:3000/components/detail/blockquote and verify that new blockquote style exists. 

https://github.com/uiowa/uids/issues/176

- Go to http://localhost:3000/components/detail/hero-block--left and verify that heading resizes smaller at a wider breakpoint. 

https://github.com/uiowa/uids/issues/175
- Removed extra spacing for when stat component is not centered. 

https://github.com/uiowa/uids/issues/185

- Go to http://localhost:3000/components/detail/header--default, http://localhost:3000/components/detail/header--site-name-with-off-canvas and http://localhost:3000/components/detail/header--department-nav-with-off-canvas and verify that mobile appears is different that desktop appearance.   Verify that it matches the screenshots below. 

<img width="552" alt="Screen Shot 2020-05-18 at 2 39 00 PM" src="https://user-images.githubusercontent.com/1036433/82252895-88ff0a00-9915-11ea-83c8-b90f060728b3.png">
<img width="387" alt="Screen Shot 2020-05-18 at 2 39 10 PM" src="https://user-images.githubusercontent.com/1036433/82252897-8997a080-9915-11ea-9988-870b925b157a.png">
<img width="380" alt="Screen Shot 2020-05-18 at 2 39 18 PM" src="https://user-images.githubusercontent.com/1036433/82252898-8997a080-9915-11ea-8c84-da90cfa826ac.png">
